db/hints/resource_manager.cc

```diff
@@ -59,8 +59,15 @@ future<semaphore_units<semaphore_default_exception_factory>> resource_manager::g
     }
     seastar::semaphore& limiter = *it->second;
```

Comment:
What's the problem with "all but one nodes simultaneously send hints to the remaining node"?

Note that 20 may be too little for small hints (bandwidth will be limited by round-trip latency) and too much for large hints (20*5MB = 100MB). Ultimately we should use rpc streams for such cases, streams eliminate the dependency on round-trip latency and place bounds on memory usage. @gleb-cloudius @asias